octobot_backtesting/data/data_file_manager.py

```diff
@@ -14,58 +14,15 @@
 #  You should have received a copy of the GNU Lesser General Public
 #  License along with this library.
 
-import time
+from datetime import datetime
 import json
-import gzip
-from os.path import basename, isfile, join, splitext
+from os.path import isfile, join, splitext
 from os import listdir, remove
 
-from octobot_commons.symbol_util import merge_currencies
-from octobot_commons.time_frame_manager import find_min_time_frame
+from octobot_commons.enums import TimeFrames
 
-from octobot_backtesting.constants import BACKTESTING_TIME_FRAMES_TO_DISPLAY, BACKTESTING_DATA_FILE_TIME_DISPLAY_FORMAT, \
-    BACKTESTING_DATA_FILE_TIME_READ_FORMAT, BACKTESTING_DATA_FILE_EXT, BACKTESTING_DATA_FILE_TIME_WRITE_FORMAT
-from octobot_backtesting.enums import DataFormatKeys, DataFormats
-
-
-def interpret_file_name(file_name):
```

Comment:
These methods are not used anymore right ?

Reply:
Not for now, we might want a similar code when coding converters from v3 backtesting data but I don't think it should be in this module anyway.